gravatar-ui/src/test/java/com/gravatar/gravatar/ui/ProfileTestUtils.kt

```diff
@@ -12,25 +18,25 @@ internal val completeProfile = emptyProfile(
     location = "Crac'h, France",
     pronouns = "They/Them",
     verifiedAccounts = listOf(
-        VerifiedAccount(
+        com.gravatar.api.models.VerifiedAccount(
```

Comment:
Would it make sense to use the named import as in other places?

Reply:
Fixed! Thanks!